### ISSUE TITLE
New Features/News/What's New Page

### ISSUE THREAD

**Issue by tim-skylight** [CDCgov/prime-simplereport#2876]
### WHY?
We have an upcoming story that will proactively alert users to new features. These alerts will be tied together with a brief explanation of a new feature. This page is where those press releases will live, as well as general product updates and share-outs.

**AS A** user of SimpleReport

**I WANT** a space where I can read product updates like new features and hear what the SimpleReport team is working on

**SO THAT** I can stay up to date on the future of SimpleReport

---

**GIVEN** I have logged into SimpleReport

**WHEN** I have selected the 'What's New' section of the application

**THEN** I am taken to a feed of updates about SimpleReport's growth and new features listed chonologically

### ANALYTICS
Would like the ability to track interactions with this page over time

### RELATED TASKS

- [ ] Add link to static page to footer of support emails

### DESIGN

**Figma**
https://www.figma.com/file/NPSPM5SBzQ7Ra3mmr9M9cB/SimpleReport-Public-Site?node-id=6883%3A25374

**Design notes and requirements**
There's currently only content made for two cards: "Send results via text or email" and "Download results". The rest of the cards on the page are present to show the future-state of the page.

Visual design details that may be easily missed:

- [ ] Gray stroke around cards
- [ ] Dropshadow on each card (probably the hardest front-end portion of this design. Happy to pair on this to explain the visual effect. I made it by adding a smaller card beneath the "New feature" card to create the floating effect. If helpful, I can try to find examples of this done on other sites in case you want to inspect their code.

**Assets/images can be found here**
https://drive.google.com/drive/folders/1WDW2ZpDO2AW-OEKeyQ-EOhCOaCD7B8px?usp=sharing

### **Present-state**
![Current state_ New features page.jpg](https://images.zenhubusercontent.com/60da5912fad056ef8402132e/8943bf78-4357-4bbb-a600-edbbf585d9d2)

### **Present-state spec**
![Current state_ Spec.jpg](https://images.zenhubusercontent.com/60da5912fad056ef8402132e/71024260-4849-40ec-bd69-0e112161d20d)

### **Future-state**
![New features page.jpg](https://images.zenhubusercontent.com/60da5912fad056ef8402132e/fde19d2c-1450-422c-be0c-7423194791b7)

### **Future-state spec**
![New features specs.jpg](https://images.zenhubusercontent.com/60da5912fad056ef8402132e/478cda9c-d5ff-409c-b052-a3bc7fde8a56)

**Comment by tim-skylight** [comment]
Step 1: build a direct path to this feature via URL while waiting on a button design
Step 2: add button to access this page

**Comment by LGFranklin** [comment]
Kenny, RLM meetnig about this week of 1/9

**Comment by Rebecca-LM** [comment]
- [Notes](https://docs.google.com/document/d/1LrquhkwC8ENRcXUgaWRGd-Byk89DvEOedsu15radEyM/edit?pli=1) on new features blog
- New features [template and source of truth doc](https://docs.google.com/document/d/166844-tbpF-poINLjQG3S0WIgiXKSaSUdhUP2fSPRcA/edit)

@tim-skylight and @LGFranklin to add info on new features to template to being process

**Comment by emmastephenson** [comment]
Kenny and Rebecca to draft first round of content here this week.

**Comment by emmastephenson** [comment]
Two new features that have content ready. @kenieh to post most up to date designs and create a new ticket for engineering to work off.

**Comment by Rebecca-LM** [comment]
@kenieh have we talked about where this will go in the static site or app navigation? @emmastephenson assuming we can put it in the app nav too, not just site?

**Comment by emmastephenson** [comment]
@kenieh - yes, would definitely like to split up into smaller tickets. Can you put some time on my cal for next week?

@Rebecca-LM - Yep, we should be able to link out to this in the app nav! And can probably make some changes to the existing nav too if that's the preference

**Comment by kenieh** [comment]
@fzhao99 to break down into the following tickets:

- Create a component for each card (image, date, title, body)
- Create a "New features" page and link as a tab under "How it works"
- Add to nav under "How it works"
- Set up analytics (page views from different sources e.g. support email, static site; click events for the card component)
- Standardize this process for adding new cards, going forward

**Comment by Rebecca-LM** [comment]
@kenieh do we want it as a tab under "How it works" or "Using SimpleReport"?

**Comment by kenieh** [comment]
@Rebecca-LM I lean towards under "How it works" for better visibility/discoverability. If I were to go for the ideal solution, I would redesign the top nav since "New features" doesn't really fit in any of the current categories. Placing it under "Using SimpleReport" makes more sense if I think about it from the User's mental model but I feel like it would lose too much visibility if we place it there. Also not sure how often people are using that "Using SimpleReport".

Do you have a preference?

**Comment by Rebecca-LM** [comment]
Lemme think about it a bit. I agree I'd like to move some stuff around on the top nav too.

**Comment by Rebecca-LM** [comment]
@kenieh @emmastephenson This will also go on the app nav, correct? In that case I think it could be better to place under "Using SimpleReport on the site which is primarily for help content, but have it be more visible on the app navigation.

**Comment by kenieh** [comment]
Mmm that makes sense to me. I'm down to go that route. CC'ing @fzhao99 for your SA.

**Comment by Rebecca-LM** [comment]
@kenieh where were you thinking to put it in the app? We can sneak in some small app nav improvements with this too. Should we chat about this?

**Comment by kenieh** [comment]
@Rebecca-LM . Probably within the "profile" icon.

Down to chat! Would love to incorporate this nav design Phoebe put together years ago.
![Screen Shot 2022-02-18 at 9 15 44 AM](https://user-images.githubusercontent.com/72819285/154730752-4a289a00-1fff-4821-ab4e-51f052b259dd.png)

**Comment by LGFranklin** [comment]
If we want to move that direction, I'd find a few admin users who switch facilities and test it. It is a much different interaction than users are familiar with and want to make sure it is intuitive. And if we do it that we figure out how to highlight it in the application.

**Comment by kenieh** [comment]
Hey @LGFranklin great suggestions for the research approach. I'll include that when we finish drafting a design ticket for improving our current nav design. This will be in the GET design backlog column.

**Comment by emmastephenson** [comment]
Design work here is finished; @fzhao99 created the engineering tickets to track this effort.
